### ISSUE TITLE
Mobile User get muted on Android once screen is locked

### ISSUE THREAD

**Issue by mekaam** [bigbluebutton/bigbluebutton#11991]
**Describe the bug**
When user join a meeting  through their Android phone at the beginning they can talk and listen but once they do not touch the screen for time to lock it then they become mute and others can not hear them till they unlock the screen , this looks like the mic go mute after few seconds of screen lock.
They still listen normal without issue during the screen lock , only the mic is affected
that only happen to Android users, but iPhone works normal when screen is lock they still can talk and listen.

**To Reproduce**
1. create a meeting with greenlight
2. join from android phone
3. screen lock after sometime of idle not touching the screen while meeting is still running
4. user can not talk in the meeting till they unlock their screen again

**Expected behavior**

**Actual behavior**

**Screenshots**

**BBB version (optional):**
BigBlueButton continually evolves. Providing the version/build helps us to pinpoint when an issue was introduced.
Example:
$ sudo bbb-conf --check | grep BigBlueButton
BigBlueButton Server 2.2.35 (2390)

**Smartphone (please complete the following information):**
 - Device: several models
 - OS: Android 
 - Browser Chrome
 - Version [e.g. 22] several versions

**Additional context**

**Comment by PlumedArcher** [comment]
I could not reproduce on Motorola G7 or G8 with OS Android using Chrome. (BBB 2.3, client  1617)

 I could still talk normally.

**Comment by mekaam** [comment]
thanks a lot for your help
may you please advise if you waited a minute or 2 while the screen is locked,
it usually stop after the lock with a minute or so

**Comment by uka-support** [comment]
@mekaam could you provide the exact devices which show the problem?

Maybe some battery optimization kicks in?

**Comment by mekaam** [comment]
Thanks @uka-support  , it looks the issue with models of xiaomi android phones

**Comment by PlumedArcher** [comment]
Yes you are right, I am sorry. When I wait one minute after the screen got locked, my mic stops as well.

**Comment by MelbaToasty** [comment]
We had similar issue on pixel 3/4a. When the phone goes to sleep/screen goes dark you loose audio/microphone.

**Comment by antobinary** [comment]
Merging into https://github.com/bigbluebutton/bigbluebutton/issues/15063
